src/main/java/example/PostgreSQLJDBC.java

```diff
@@ -0,0 +1,50 @@
+package example;
+
+import database.DataInstancesBuilder;
+import database.DatabaseConnection;
+
+import java.sql.*;
+
+import java.sql.Connection;
+import java.sql.Statement;
+
+/**
+ * Created by tlfrd on 29/10/2015.
+ */
+public class PostgreSQLJDBC {
+    public static void main(String args[]) {
+        Connection c = null;
+        Statement stmt = null;
+        try {
+            c = DatabaseConnection.getDatabaseConnection();
+            System.out.println("Opened database successfully");
```

Comment:
Set up a Logger so we don't have tons of different information printed to stdout.